app/services/audit_results.rb

```diff
@@ -92,6 +91,7 @@ class AuditResults
     UNEXPECTED_VERSION
     # Temporary fix for workflow-service throwing exceptions
     # because some error reports from MoabReplicationAudit are too long
+    # INVALID_MANIFEST,
```

Comment:
my commit message said INVALID_MOAB;  I went with the code itself rather than the commit message.  I have no idea what I was thinking -- too long ago.  I am happy to move INVALID_MANIFEST back to the list.

Reply:
yeah, let's do that if you can't recall a concrete reason for the move.

Reply:
INVALID_MANIFEST is back in the list.